src/FeedRegistration.php

```diff
@@ -47,7 +47,15 @@ class FeedRegistration {
 	public function __construct( $local_feeds_configurations, $feed_generator ) {
 		$this->configurations = $local_feeds_configurations;
 		$this->feed_generator = $feed_generator;
+		$this->init();
```

Comment:
Moving the hooks to a separate method is 👍🏻 

But the hooks are still called every time you call `new FeedRegistration()`. The `init(()` method should be called separately from the constructor. Just like `$feed_generator` here:

https://github.com/woocommerce/pinterest-for-woocommerce/blob/28248c5f5a3e99b2a906df84886c80be6b238cc2/src/ProductSync.php#L107-L113

Reply:
totally missed that :/